localstack-core/localstack/aws/handlers/validation.py

```diff
@@ -92,6 +95,7 @@ def __call__(self, chain: HandlerChain, context: RequestContext, response: Respo
         if not config.OPENAPI_VALIDATE_REQUEST:
             return
 
+        hasattr(self, "open_apis") or self._load_specs()
```

Comment:
It might be cleaner to move the conditional into `_load_specs()` but yes, it will impact the performance due to the extra call